depthai_ros_driver/include/depthai_ros_driver/device_ros.hpp

```diff
@@ -135,16 +157,21 @@ class DeviceROS : public dai::DeviceBase {
                 case dai::DatatypeEnum::Buffer:
                     break;
                 case dai::DatatypeEnum::CameraControl:
+                    _pub_t["CameraControl"] = std::thread{generate_pub_lambda<depthai_datatype_msgs::RawCameraControl>(_pub_nh, "CameraControl", 10)};
```

Comment:
Let's use the stream name instead of this to allow for multiple in/out puts of the same type